editor/src/clj/editor/tile_source.clj

```diff
@@ -688,32 +699,49 @@
+                                                (not= img-h coll-h))
+                                        (g/error-fatal (format "both 'Image' and 'Collision' must have the same dimensions (%dx%d vs %dx%d)"
+                                                               img-w img-h
+                                                               coll-w coll-h)))))))
   (output tile-width-error g/Err (g/fnk [image-size collision-size tile-width tile-margin]
-                                        (let [dims (or image-size collision-size)]
-                                          (when dims
-                                            (let [{w :width} dims
-                                                  total-w (+ tile-width tile-margin)]
-                                              (when (< w total-w)
-                                                (g/error-fatal (format "the total width ('Tile Width' + 'Tile Margin') is greater than the 'Image' width (%d vs %d)"
-                                                                       total-w w))))))))
+                                   (let [dims (or image-size collision-size)]
+                                     (when dims
+                                       (let [{w :width} dims
+                                             total-w (+ tile-width tile-margin)]
+                                         (when (< w total-w)
+                                           (g/error-fatal (format "the total width ('Tile Width' + 'Tile Margin') is greater than the 'Image' width (%d vs %d)"
+                                                                  total-w w))))))))
   (output tile-height-error g/Err (g/fnk [image-size collision-size tile-height tile-margin]
-                                         (let [dims (or image-size collision-size)]
-                                           (when dims
-                                             (let [{h :height} dims
-                                                   total-h (+ tile-height tile-margin)]
-                                               (when (< h total-h)
-                                                 (g/error-fatal (format "the total height ('Tile Height' + 'Tile Margin') is greater than the 'Image' height (%d vs %d)"
-                                                                        total-h h)))))))))
+                                    (let [dims (or image-size collision-size)]
+                                      (when dims
+                                        (let [{h :height} dims
+                                              total-h (+ tile-height tile-margin)]
+                                          (when (< h total-h)
+                                            (g/error-fatal (format "the total height ('Tile Height' + 'Tile Margin') is greater than the 'Image' height (%d vs %d)"
+                                                                   total-h h)))))))))
```

Comment:
Just an indentation here.

Reply:
I suggest we take this opportunity to move the entire `g/fnk` expression to its own line below the `output` line to decrease indentation a bit.